install.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/sh
+
+# Install git-lfs for OSX
+if [[ "$OSTYPE" =~ ^darwin ]]; then
+  if command -v brew
+  then
+    brew install git-lfs
+    git lfs install
```

Comment:
We might want to check whether `git-lfs` is already installed, instead of running the install command by default – running a brew install command will cause brew to run `brew update --preinstall`, which is very slow depending on the last time you've updated brew.

Reply:
done